### DISCUSSION TITLE
Support for Tangnano boards

### DISCUSSION THREAD

**Discussion by Greeniac** [FPGAwars/icestudio#825] [New Boards Requests]

**Comment by cavearr** [comment]
We are currently working on it, very soon the support will be done.

**Comment by bsom** [comment]
This would be AWESOME as their current toolchain is clumsy.

**Comment by cavearr** [comment]
Hi @bsom! sorry for my delay in this task, i'm very close to finish this integration but other tasks take me more time that i had planed.

I hope in the next two or three weeks as much we have this boards running.

Thanks again folks for use Icestudio!!

**Comment by cavearr** [comment]
Hi my friends! fresh news around this, now i have working some sipeed boards, around a couple of weeks i'm telling you to try it, sorry for the delay in this task.

**Comment by cavearr** [comment]
Hello! I am reorganizing the issues, trying to give more visibility and be able to better organize and prioritize the work and my roadmap (I will publish it very soon), I am converting some issues that are **new feature requests** as **ideas** on Github **discussions**.

Thanks for understanding.

**Comment by ellisgl** [comment]
Just wondering where in the list of things does the fall at? I would love to mess around with it on 9Ks and 20K (also have a 138l console)

**Comment by TunayAdaKaracan** [comment]
Any updates on this yet?

**Comment by cavearr** [comment]
I'm preparing a new huge version of Icestudio for the the first week of October, in this release this boards should be included.
